### DISCUSSION TITLE
What happened to the Golden Edition?

### DISCUSSION THREAD

**Discussion by CypherPotato** [usebruno/bruno#3473] [Q&A] [answered]
I was interested in purchasing the Bruno Golden Edition, to support the project, and because it is a permanent purchase, but I saw that it is no longer on the /pricing page on the website.

What there was? Has it been discontinued and now there are only monthly plans?

**Comment by CypherPotato** [comment]
That's horrible.

It goes against the reason the project was born: to be different from Insomnia and Postman.

Now it's becoming a subscription service. Bruno sold a plan called "lifetime" and they won't even keep it anymore. This is disrespectful to whoever bought it. I was going to support the project because I saw this license as support to maintain the good product, not to be held hostage by yet another API testing service.

Now, I shall start to look for an alternative, once again...

**Reply by DaniBedz** [reply]
Agreed. Was just about to pull the trigger on the golden edition. Will look elsewhere now, I won't support a product with a subscription model.

**Reply by CypherPotato** [reply]
@DaniBedz The [restfox](https://restfox.dev/) project was my go-to API test. It's an Insomnia fork without subscription bullshit and git support.

**Reply by DaniBedz** [reply]
This is great, importing from an open api url works better than any alternatives I've tried.

Thank you.

**Reply by jikuja** [reply]
> Now it's becoming a subscription service. Bruno sold a plan called "lifetime" and they won't even keep it anymore. This is disrespectful to whoever bought it. 

Indeed. I have serious doubts if the availability/current price of the subscription can be trusted or not.

I wonder if organizations even ends up buying subscriptions after reading the documentation about secret encryption.

**Reply by Nexz** [reply]
Weighing in here, was using Bruno a lot these days and wanted to chip in... absolutely not going to subscribe to a desktop application. I suggest following the JetBrains route where you purchase a certain version and if you need upgrades/updates, you can subscribe to get a newer version.

**Reply by helloanoop** [reply, marked as answer]
> Bruno sold a plan called "lifetime" and they won't even keep it anymore. This is disrespectful to whoever bought it.

@CypherPotato If you have brought a license, you can use it for lifetime. The changes apply only to new purchases.

@Nexz I've explained our stance and reasoning in https://github.com/usebruno/bruno/discussions/3486#discussioncomment-12699109
